kiosk/src/Models/Kiosk.ts

```diff
@@ -79,29 +79,36 @@ export class Kiosk {
         return desc;
     }
 
-    async saveNewGameAsync(gameId: string): Promise<GameData | undefined> {
+    async saveNewGameAsync(gameIds: string[]): Promise<GameData[]> {
```

Comment:
nit: should you rename this to reflect that it's now possibly more than one game? Like `saveNewGamesAsync`